### DISCUSSION TITLE
How to create gradient Text / Heading?

### DISCUSSION THREAD

**Discussion by iUnknownAdorn** [webstudio-is/webstudio-community#92] [Q&A] [answered]
I am unable to create gradient text / heading.
I am not talking about gradient background - it works.

But for gradient Text, there is no option to add gradient color to the text, it seems i can only add RGBA color.

![Screenshot 2024-01-02 at 8 34 19 AM](https://github.com/webstudio-is/webstudio-community/assets/8273366/57e9da04-c389-43a9-b341-044b10524162)

**Comment by kof** [comment, marked as answer]
Gradient can only be added using background image gradient in CSS.
Color has to be transparent
